modules/integration/tests-integration/tests-amqp/src/test/java/org/wso2/mb/integration/tests/amqp/functional/DurableTopicServerRestartTestCase.java

```diff
@@ -96,13 +97,13 @@ public void performDurablePublishRestartServerTestCase()
         AndesClient publisherClient = new AndesClient(publisherConfig, true);
         publisherClient.startClient();
 
-        AndesClientUtils.sleepForInterval(15000L);
+        AndesClientUtils.sleepForInterval(5000L);
 
         // Restarting the server
         super.restartServer();
 
         // Waiting till server is completely started
-        AndesClientUtils.sleepForInterval(60L * 1000L);
+        AndesClientUtils.sleepForInterval(20L * 1000L);
```

Comment:
Maybe we don't need this sleep since test framework itself waits until the server restarts when restartServer() is called.